src/consensus/tx_verify.cpp

```diff
@@ -126,7 +126,9 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& in
     unsigned int nSigOps = 0;
```

Comment:
I figured it wasn't worth it - if we hit those errors in consensus code, we'd better crash cause we might fall out of consensus...if we accidentally add something to our mempool that should be out of policy...oh well, not a big deal.